linkedlist.py

```diff
@@ -0,0 +1,86 @@
+class Node:
+    def __init__(self, data=None, next=None):
+        self.data = data
+        self.next = next
+
+
+class LinkedList:
+    def __init__(self):
+        self.head = None
+
+    def show_list(self):
+        if self.head == None:
```

Comment:
```suggestion
        if self.head is None:
```

Do porownan z none powinnas uzywac `is` i `is not`
https://stackoverflow.com/a/14247383